### ISSUE TITLE
Auto update on packagist

### ISSUE THREAD

**Issue by gianarb** [influxdata/influxdb-php#45]
Hello @thecodeassassin do you have permission to add the packagist's token in order to sync in automatic our update on packagist?

@virgofx thanks for your ping

Thanks a lot 
PS at the moment the 1.4.0 is not up to date on packagist

**Comment by virgofx** [comment]
:+1:

**Comment by virgofx** [comment]
ping @thecodeassassin @gianarb 

Any way we can get this auto-updated (still having to use custom repository until packagist updates).

**Comment by thecodeassassin** [comment]
@virgofx i am currently on vacation but will take care of this as soon as possible. sorry for the late reply

**Comment by thecodeassassin** [comment]
Autoupdate was already enabled, i refreshed the token (maybe that was causing a problem).

**Comment by virgofx** [comment]
Weird, still shows only the 1.4 version. Must be something going on. I believe you need to have the correct username and token from your Packagist profile. Ensure the github.com setting also has enabled. Then head back to packagist and maybe try to update ...

![image](https://cloud.githubusercontent.com/assets/739719/15577993/9186a0d8-2313-11e6-910c-83eb131ec5eb.png)

![image](https://cloud.githubusercontent.com/assets/739719/15578002/9933130c-2313-11e6-807e-42d42ac60825.png)

**Comment by thecodeassassin** [comment]
@virgofx exactly, i have no idea why it doesn't work. The github service says the payload was delivered successfully :S

**Comment by tuupola** [comment]
While you are at it, try clicking the green update button in Packagist. I am keen to test the new version.

**Comment by tuupola** [comment]
What are you using as username in GitHub packagist hook settings?

**Comment by thecodeassassin** [comment]
@tuupola the exact username i use on packagist.

**Comment by thecodeassassin** [comment]
@tuupola updated

**Comment by tuupola** [comment]
Try changing the GitHub repository address to current one. Currently Packagist has `https://github.com/influxdb/influxdb-php` but it has been changed at some point to `https://github.com/influxdata/influxdb-php`. Maybe this confuses Packagist.

**Comment by virgofx** [comment]
Also the composer.json needs to have the **name** updated as well. New packages don't work unless they match properly.

**Comment by virgofx** [comment]
I would leave existing packagist as is. Then add a newone with proper name, marked as deprecated in readme. This will give people time to switch composer updates.

**Comment by tuupola** [comment]
The name has not been changed. Only repository address. Composer name and repository do not need to match.

**Comment by thatside** [comment]
Hello!

Please consider changing package name on Packagist according to new Github name (see https://github.com/composer/packagist/issues/47).
Current naming doesn't allow to use forks of this repo as a replacement of master repo for dependant packages (like https://github.com/Algatux/influxdb-bundle).
These packages depend on influxdb/influxdb-php, but forked packages automatically become influxdata/influxdb-php based on composer.json info.
